src/pages/workspace/WorkspacePeoplePage.js

```diff
@@ -0,0 +1,328 @@
+                    confirmText={this.props.translate('common.remove')}
+                    cancelText={this.props.translate('common.cancel')}
+                />
+                <ScrollView style={[styles.settingsPageBackground]} bounces={false}>
+                    <View style={styles.pageWrapper}>
+                        <Text style={[styles.mb6, styles.textP, styles.textFull]}>
+                            {this.props.translate('workspace.people.tagline')}
+                        </Text>
+                        <View style={styles.buttonRow}>
+                            <Button
+                                success
+                                style={[]}
+                                text={this.props.translate('common.invite')}
+                                onPress={this.inviteUser}
+                            />
+                            <Button
+                                danger
+                                style={[styles.ml2]}
+                                isDisabled={this.state.selectedEmployees.length === 0}
+                                text={this.props.translate('common.remove')}
+                                onPress={this.askForConfirmationToRemove}
+                            />
+                        </View>
+                        <View style={[styles.w100, styles.mt4]}>
+                            {
+                                this.props.policy.employeeList ? (
+                                    <FlatList
+                                        ListHeaderComponent={this.renderHeader()}
+                                        ref={flatListRef}
+                                        renderItem={this.renderItem}
+                                        contentContainerStyle={[styles.w100]}
+                                        initialRowHeight={32}
+                                        // eslint-disable-next-line max-len
+                                        data={(this.props.policy.employeeList && this.props.policy.employeeList.length !== 0)
+                                            // eslint-disable-next-line max-len
+                                            ? this.props.policy.employeeList.map(email => this.props.personalDetails[email] || false)
+                                            : []}
+                                        maxToRenderPerBatch={1}
+                                        windowSize={15}
+                                        removeClippedSubviews={this.props.shouldRemoveClippedSubviews}
+                                    />
+                                ) : (
+                                    <View style={[styles.peopleRow]}>
+                                        <View style={[styles.peopleRowCell, styles.flex2]}>
+                                            <Text>
+                                                Unfortunately, this workspace has no members.
```

Comment:
1. I think maybe it's not possible since we should always have at least one member
2. If not, did we get this copy approved or where did it come from?
3. Why so serious?? 😂

Reply:
1. You are right, I made this edge case without thinking about it too much.
2. Nope, that was just a placeholder.
3. 🥺 coz I dont want to mess up in my new job.